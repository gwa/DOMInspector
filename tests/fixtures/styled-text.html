<article>
    <p>
        This is some <strong>text</strong> with <em>inline styles</em>
        and a <a href="http://www.example.com">link</a>.<BR/>
        With a line break.
    </p>
    <p>
        A second paragraph.
    </p>
</article>
