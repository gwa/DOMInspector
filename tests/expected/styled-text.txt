This is some text with inline styles and a link.
With a line break.

A second paragraph.
